docker/openproblems-r-extras/Dockerfile

```diff
@@ -1,19 +1,29 @@
 FROM singlecellopenproblems/openproblems-r-base:latest
 
+
+RUN apt-get update
+RUN apt-get install -y libhdf5-dev
+
 ARG NB_USER="sagemaker-user"
 ARG NB_UID="1000"
 ARG NB_GID="100"
 
 USER root
 WORKDIR /
 
+
 # Install R packages
 COPY ./docker/openproblems-r-extras/r_requirements.txt ./r_requirements.txt
 RUN R -e "renv::install(scan('r_requirements.txt', sep='\n', what=character()))"
+RUN R -e 'install.packages("devtools")'
+RUN R -e 'devtools::install_github("immunogenomics/harmony")'
+RUN R -e 'devtools::install_github("welch-lab/liger@v0.5.0")'
+
```

Comment:
just a note for the future, we should at some point clean this up to an R install script.